packages/scaffold-ui/test/partials/w3m-header.test.ts

```diff
@@ -0,0 +1,233 @@
+import { elementUpdated, fixture } from '@open-wc/testing'
+import { beforeAll, beforeEach, describe, expect, it, vi } from 'vitest'
+
+import { html } from 'lit'
+
+import type { CaipNetwork } from '@reown/appkit-common'
+import {
+  AssetController,
+  AssetUtil,
+  ChainController,
+  EventsController,
+  ModalController,
+  OptionsController,
+  RouterController,
+  SIWXUtil
+} from '@reown/appkit-core'
+
+import { W3mHeader } from '../../src/partials/w3m-header'
+import { HelpersUtil } from '../utils/HelpersUtil'
+
+describe('W3mHeader', () => {
+  let element: W3mHeader
+
+  beforeAll(() => {
+    Element.prototype.animate = vi.fn().mockReturnValue({ finished: Promise.resolve() })
+  })
+
+  beforeEach(async () => {
+    vi.clearAllMocks()
+    RouterController.reset('Connect')
+    element = await fixture(html`<w3m-header></w3m-header>`)
+    await element.updateComplete
+  })
+
+  describe('Network Selection', () => {
+    const mockNetwork = {
```

Comment:
Can we use networks from our exports instead of mocking? If we need extended one, we can use CaipNetworkUtil.extendCaipNetwork